modules/widgets/google-translate.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Plugin Name: Google Translate Widget for WordPress.com
+ * Plugin URI: http://automattic.com
+ * Description: Add a widget for automatic translation
+ * Author: Artur Piszek
+ * Version: 0.1
+ * Author URI: http://automattic.com
+ * Text Domain: jetpack
+ */
+
+class Google_Translate_Widget extends WP_Widget {
+	static $instance = null;
+
+	/**
+	 * Register widget with WordPress.
+	 */
+	function __construct() {
+		parent::__construct(
+			'google_translate_widget',
+			/** This filter is documented in modules/widgets/facebook-likebox.php */
+			apply_filters( 'jetpack_widget_name', __( 'Google Translate', 'jetpack' ) ),
+			array( 'description' => __( 'Automatic translation of your site content', 'jetpack' ) )
+		);
+		wp_register_script( 'google-translate-init', plugins_url( 'google-translate/google-translate.js', __FILE__ ) );
+		wp_register_script( 'google-translate', '//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit', [ 'google-translate-init' ] );
+	}
+
+	/**
+	 * Display the Widget.
+	 *
+	 * @see WP_Widget::widget()
+	 *
+	 * @param array $args     Display arguments.
+	 * @param array $instance The settings for the particular instance of the widget.
+	 */
+	public function widget( $args, $instance ) {
+		// We never should show more than 1 instance of this.
+		if ( null === self::$instance ) {
+			/** This filter is documented in core/src/wp-includes/default-widgets.php */
+			$title = apply_filters( 'widget_title', $instance['title'] );
```

Comment:
In Customizer preview, this throws an warning
```
 Notice: Undefined index: title in /wp-content/plugins/jetpack/modules/widgets/google-translate.php on line 41
```
Please check if it's set before using it `isset( $instance['title'] ) ? $instance['title'] : ''`